### DISCUSSION TITLE
Is the measurement function supposed to add padding+border?

### DISCUSSION THREAD

**Discussion by juancampa** [DioxusLabs/taffy#623] [General]
https://github.com/DioxusLabs/taffy/assets/1410520/df8a3f7e-361c-4b5c-8c7a-d343cf8bd320

Notice the title text (to the right of the pink number), the inner blue rect is the padded rect (`layout.size` shrunk by the 20px in `layout.padding`). When shrinking the window, the rendered text wraps before the measured text wraps. The reason seems to be that when measuring, the available_size does not include padding.

Is the measure function supposed to add padding+border to the measured text size? I don't see that being done in the examples so maybe I'm doing something else wrong.

**Comment by juancampa** [comment]
As I keep working on this, the issue seems to be that the padding/border passed in `taffy.new_leaf` is not taken into account while measuring.

**Reply by nicoburns** [reply]
This has been fixed (or changed, depending on whether you consider the old behaviour a bug or not) in the latest Taffy release (v0.4)
